manifests/03-rbac-rolebinding.yaml

```diff
@@ -11,6 +11,20 @@ subjects:
 - kind: ServiceAccount
   name: console-operator
 ---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
```

Comment:
Pondering this as well, It's not immediately clear why the addition.  Quick comment to clarify? Thanks!

Reply:
We need to be able to read existing `configmap/extension-apiserver-authentication` in *kube-system* namespace. 
*extension-apiserver-authentication* is the name of ConfigMap in the kube-system namespace holding the root certificate bundle to use to verify client certificates on incoming requests before trusting usernames in headers specified by --requestheader-username-headers. This is created in the cluster by the kube-apiserver.
Explained [here](https://github.com/deads2k/kubernetes/blob/1105e4e0d1c0e78e2a203a136e9f8bcaff5c36ab/staging/src/k8s.io/apiserver/pkg/server/options/authentication.go#L235-L239).

Reply:
Turn off the delegated authn/authz (it is useless to you) and drop this RBAC change.